reference/release-notes/1.23.0.md

```diff
@@ -0,0 +1,92 @@
+
+### Full support for AAOS
+
+Android Automotive OS based images are fully supported starting from this release and no longer considered an experimental feature.<!--AC-2660-->
+
+### Anbox Cloud dashboard
+
+The Anbox Cloud dashboard has the following improvements:
+
+  * The *Instances* and the *Sessions* pages and their functionalities are merged to provide an enhanced user experience while streaming.
+
+  * The initial VHAL values upon joining a stream are now captured as a default. You can revert to these values using the *Reset* button, when needed.
+
+  * You can set up and authorise ADB connections for an instance from the dashboard.
+
+### Anbox Cloud charms
+
+All Anbox Cloud charms now install Snaps directly from the Snap Store instead of using deb packages. Existing installations automatically switch to snap refresh from the store on upgrade.<!--AC-2523-->
+
+### Vendor image
+
+The Anbox vendor images are now based on Android 14. Building on the latest version of Android helps us provide the latest updates and security fixes.
+
+### Security updates
+
+Android security updates for August 2024 (see [Android Security Bulletin - August 2024](https://source.android.com/docs/security/bulletin/2024-08-01) for more information).
+
+### WebView updates
+
+The Android WebView has been updated to [127.0.6533.84](https://chromereleases.googleblog.com/2024/07/chrome-for-android-update_30.html).
+
+### Documentation
+
+Our documentation is now hosted at [documentation.ubuntu.com](https://documentation.ubuntu.com/anbox-cloud/en/latest). Previous documentation links from the [earlier website](https://anbox-cloud.io) will be redirected to new targets.
+
+The documentation source is hosted at the public [Anbox Cloud documentation repository on GitHub](https://github.com/canonical/anbox-cloud-docs). We welcome all contributions to the Anbox Cloud documentation using this repository. See [Contribute to Anbox Cloud documentation](https://documentation.ubuntu.com/anbox-cloud/en/latest/contribute/landing/) for more information.
+
+## Removed functionality
+
+Support for EmuGL is removed with this release.
+
+## Deprecations
+
+Use of a node controller to manage port forwarding from an instance is deprecated, starting with this release. In 1.23.0, the node controller will still be used for the full Anbox Cloud deployment. For the appliance, the Anbox Management Service (AMS) now uses a LXD proxy device to manage the port forwarding from an instance to expose a service externally or internally.
+
+In the 1.24.0 release, the node controller will be fully replaced with LXD proxy device for both regular and appliance deployments.<!--AC--2487-->
+
+## Known issues
+
+* WiFi support is not available with any of the images. This issue will be fixed in the 1.23.1 release. <!--AC-2647-->
+* No crash dump is generated for the `anbox session` or `container-manager` process when running an Anbox session on ARM64.
+
```

Comment:
```suggestion
* In some rare cases, the VHAL panel will not show up on AAOS streams. Reloading the stream page fixes the issue in most cases. In case it doesn't, restarting the instance will fix the issue.

```
@keirthana Feel free to ask for me for changes, or do changes yourself if you feel like this should be written differently.

Reply:
This is perfect, thanks!